www/templates/intro/summary.html

```diff
@@ -1,14 +1,12 @@
 <ion-content class="has-footer">
 	<!-- <div class="intro-space"></div> -->
 	<div class="intro-title">
-	<center><h4>E-Mission: Data driven carbon emission reduction</h4></center>
+	<center><h4>CCI: Assessing the Travel Demand and Co-Benefit Impacts of Affordable Transit-Oriented Development</h4></center>
 	</div>
 	<!-- <div class="intro-space"></div> -->
 	<div class="intro-text">
-	This version of E-mission is designed to:
 	<div class="intro-space"></div>
-	<li> This version of E-Mission is designed to record your travel behavior for a period of four days in order to calculate your Vehicle Miles Travelled (VMT). The application will collect trip location data, travel speed, and estimate mode of travel such as walking, biking, or vehicle. All data collected is used solely for the purpose of research for the research study at UC Berkeley’s Center for Community Innovation, entitled “Assessing the Travel Demand and Co-Benefit Impacts of Affordable Transit-Oriented Development.”
-	</ol>
+	This version of E-Mission is designed to record your travel behavior for a period of four days in order to calculate your Vehicle Miles Travelled (VMT). The application will collect trip location data, travel speed, and estimate mode of travel such as walking, biking, or vehicle. All data collected is used solely for the purpose of research for the research study at UC Berkeley’s Center for Community Innovation, entitled “Assessing the Travel Demand and Co-Benefit Impacts of Affordable Transit-Oriented Development.”
```

Comment:
This is not a code change, but the summary says four days and the consent says one week